moveit_experimental/moveit_jog_arm/CMakeLists.txt

```
@@ -37,27 +37,27 @@ include_directories(
   ${EIGEN3_INCLUDE_DIRS}
 )
 
-add_executable(${PROJECT_NAME}_server
-  src/jog_arm/collision_check_thread.cpp
-  src/jog_arm/jog_arm_server.cpp
-  src/jog_arm/jog_calcs.cpp
-  src/jog_arm/jog_ros_interface.cpp
-  src/jog_arm/low_pass_filter.cpp
+add_executable(jog_arm_server
+  src/moveit_jog_arm/collision_check_thread.cpp
```

I suggest to skip this extra folder layer:
```suggestion
  src/collision_check_thread.cpp
```